charts/gatekeeper/crds/assign-customresourcedefinition.yaml

```diff
@@ -0,0 +1,245 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
CRDs should only be deployed if mutation flag is enabled

(this change should be in helmify and should modify manifest_staging/charts)